### PR TITLE
fix[opentelemetry-kube-stack] add persistentvolumeclaims to ClusterRole

### PR DESCRIPTION
**Pull request by genadipost**
fixes https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1444

### CODE REVIEW

**Review by jaronoff97 — COMMENTED** [review]
thank you for your contribution! can you bump the patch version of the chart and run `make generate-examples CHARTS=opentelemetry-kube-stack` when you get a chance?

**Comment by genadipost** [comment]
@jaronoff97 There is some tool or make command to perform the bump?

**Comment by jaronoff97** [comment]
Yep, there's a makefile at the root of the repo. I posted the command above :)

**Comment by genadipost** [comment]
> You should also add `persistentvolumes` as well

I can make another pull request

**Comment by genadipost** [comment]
@povilasv @jaronoff97 I have bumped `make generate-examples CHARTS=opentelemetry-kube-stack`

**Comment by genadipost** [comment]
> @genadipost you need to bump the patch version and then run `make generate-examples CHARTS=opentelemetry-kube-stack`

Done :)

File: charts/opentelemetry-kube-stack/templates/clusterrole.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ rules:
   - pods
   - events
   - secrets
+  - persistentvolumeclaims
```
Comment (TylerHelmuth) [inline charts/opentelemetry-kube-stack/templates/clusterrole.yaml:20]: @jaronoff97 is this another case where the operator needs the same permissions as the collector's receivers?

Comment (jaronoff97) [reply charts/opentelemetry-kube-stack/templates/clusterrole.yaml:20]: I believe so, yes

Comment (genadipost) [reply charts/opentelemetry-kube-stack/templates/clusterrole.yaml:20]: @jaronoff97 @TylerHelmuth Can this be resolved?

**Comment by jaronoff97** [comment]
@genadipost when you get a chance, could you sign the CLA please?

**Comment by genadipost** [comment]
> @genadipost when you get a chance, could you sign the CLA please?

@jaronoff97 Just did.

**Review by jaronoff97 — APPROVED** [review]

**Pull request merged**